docs/configuration/vpn/wireguard.rst

```diff
@@ -10,10 +8,24 @@ WireGuard is an extremely simple yet fast and modern VPN that utilizes
 state-of-the-art cryptography. See https://www.wireguard.com for more
 information.
 
+*************
```

Comment:
the headline over and/or underlines must be the same length as the headline. New sphinx versions will ignore this, but in read the docs will raise build warnings about this.